emotion_ferplus/README.md

```diff
@@ -33,12 +33,30 @@ import numpy as np
 from PIL import Image
 
 def preprocess(image_path):
+  # model input shape
   input_shape = (1, 1, 64, 64)
+  # load image and resize
   img = Image.open(image_path)
```

Comment:
You can convert  to gray level by calling convert() on Image object no need to iterate.: img = Image.open('image.png').convert('L')